instrumentation/kamon-elasticsearch/src/main/resources/reference.conf

```diff
@@ -0,0 +1,23 @@
+# ================================================== #
+# kamon Elasticsearch client reference configuration #
+# ================================================== #
+
+kamon.instrumentation.elasticsearch {
+}
+
+kanela {
+  modules {
+    elasticsearch-driver {
+
+      name = "Elasticsearch client"
+      description = "Elasticsearch Client Instrumentation"
```

Comment:
Name: Elasticsearch Client
Description: Provides tracing of client calls made with the official Elasticsearch Client library.